nano/node/rpc.cpp

```diff
@@ -1391,8 +1404,10 @@ void nano::rpc_handler::bootstrap_status ()
 
 void nano::rpc_handler::chain (bool successors)
 {
+	successors = successors || request.get<bool> ("reverse", false);
```

Comment:
Maybe this should be XOR instead of OR ?

RPC successors(reverse=true) == chain
RPC chain(reverse=true) == successors

`chain(true)`, reverse=false == successors
`chain(true)`, reverse=true == chain
`chain(false)`, reverse=false == chain
`chain(false)`, reverse=true == successors

| successors | XOR | reverse | = | successors' |
|:----------:|:---:|:-------:|:-:|:------:|
|    true    |  ^  |  false  | = |  true  |
|    true    |  ^  |   true  | = |  false |
|    false   |  ^  |  false  | = |  false |
|    false   |  ^  |   true  | = |  true  |

Reply:
I thought about it. Well, if you think it's logical, let's do it

Reply:
Not to bike-shed, but I think `successors = successors != request.get<bool> ("reverse", false);` is clearer when reading that code down the road.